src/components/application_manager/include/application_manager/resumption/resumption_data_json.h

```diff
@@ -216,13 +213,13 @@ class ResumptionDataJson : public ResumptionData {
    * @brief Set applications for resumption to LastState
    * @parems apps_json applications to write in LastState
    */
-  void SetSavedApplication(Json::Value& apps_json);
+  void SetSavedApplication(Json::Value& apps_json, Json::Value& dictionary);
 
   /**
    * @brief Setup IgnOff time to LastState
    * @param ign_off_time - igition off time
    */
-  void SetLastIgnOffTime(time_t ign_off_time);
+  void SetLastIgnOffTime(time_t ign_off_time, Json::Value& dictionary);
```

Comment:
@YarikMamykin please provide description for all newly added params in this header

Reply:
@AKalinich-Luxoft fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/3dcb90541c1ee0310a396d2564a95eba3e12237e